controller/src/beerocks/master/db/db.cpp

```diff
@@ -4431,14 +4398,18 @@ bool db::remove_client_entry_and_update_counter(const std::string &entry_name)
     }
     --m_persistent_db_clients_count;
 
+    LOG(DEBUG) << "client client entry " << entry_name
+               << " from persistent db, total clients count in persisttent-db: "
+               << m_persistent_db_clients_count;
+
     return true;
 }
 
-bool db::remove_candidate_client()
+bool db::remove_candidate_client(sMacAddr client_to_skip)
```

Comment:
```suggestion
bool db::remove_candidate_client(const sMacAddr &client_to_skip)
```

Reply:
you cannot provide a const reference and a default value